quantities/Transformation.cs

```diff
@@ -0,0 +1,69 @@
+using Quantities.Numerics;
+
+namespace Quantities;
+
+public sealed class Transformation
+{
+    private Double nominator, denominator, offset;
+    internal Transformation()
+    {
+        this.offset = 0d;
+        this.nominator = this.denominator = 1d;
+    }
+    private Transformation(in Double nominator, in Double denominator, in Double offset)
+    {
+        this.nominator = nominator;
+        this.denominator = denominator;
+        this.offset = offset;
+    }
+    private Transformation Add(in Double value)
+    {
+        this.offset += value;
+        return this;
+    }
+    private Transformation Subtract(in Double value)
+    {
+        this.offset -= value;
+        return this;
+    }
+    private Transformation Multiply(in Double value)
+    {
+        this.offset *= value;
+        this.nominator *= value;
+        return this;
+    }
+    private Transformation Divide(in Double value)
+    {
+        this.offset /= value;
+        this.denominator *= value;
+        return this;
+    }
+    internal Transformation Pow(Int32 exponent)
```

Comment:
Mathematically speaking this `Pow` function is not implemented correctly. But for all quantities that require this function, it's just right.

Importantly: We can change this without introducing any braking changes, as it's an `internal` implementation detail on the 'Transformation' class.

Reply:
I don't think making it nullable adds value as the value `0` already serves that purpose. I will throw a `NotImplemented` exception though. As that makes it explicit what's happening.

The naming comes from the [`Pow`](https://learn.microsoft.com/en-us/dotnet/api/system.math.pow) function on `System.Math`.

Reply:
Implemented the check for zero. Thanks for the comment!

Reply:
I'm glad you found the comment useful. Actually, what I thought about is an making the offset as on optional parameter. And then, if the offset is set, you'd throw the `NotImplemented` exception.

However, this is just a minor detail which is probably not worth talking about :)